### PR TITLE
(GH-1111) Modified pro business messages to sound more professional

### PR DESCRIPTION
**Pull request by birgersp**
Following the discussion in chocolatey/choco##1111, pro business messages
are modified to what I deem more appropriate

### CODE REVIEW

File: src/chocolatey/infrastructure.app/services/ChocolateyPackageService.cs [hunk 1]
```diff
@@ -65,7 +65,7 @@ public class ChocolateyPackageService : IChocolateyPackageService
 @"
 Did you know the proceeds of Pro (and some proceeds from other 
  licensed editions) go into bettering the community infrastructure?
- Your support ensures an active community, it makes you look smarter,
+ Your support ensures an active community, keeping Chocolatey tip top,
```
Comment (gep13) [inline src/chocolatey/infrastructure.app/services/ChocolateyPackageService.cs]: @birgersp I agree that `it makes you look smarter` is a candidate for replacement, but not sure if `keeping Chocolatey tip top` fits the bill.  If nothing else, I think it should be `keeps Chocolatey tip top` in order to better match the context of the sentence.  Thoughts?

Comment (ferventcoder) [inline src/chocolatey/infrastructure.app/services/ChocolateyPackageService.cs]: Both work for me.

Comment (birgersp) [inline src/chocolatey/infrastructure.app/services/ChocolateyPackageService.cs]: I concur, I'll fix it asap

**Comment by ferventcoder** [comment]
@birgersp while most documentation type changes wouldn't need a CLA signature, in this case these messages are considered marketing so a CLA signature is desired. If you can sign that, we can get this pulled in. Thanks!

**Comment by ferventcoder** [comment]
Thanks!

**Comment by ferventcoder** [comment]
Merged into stable at https://github.com/chocolatey/choco/commit/775c5f7563605454a37cac7fca255f129c2adf81

**Pull request closed without merging**